lib/process-services-cloud/src/lib/task/task-form/components/task-form-cloud.component.ts

```diff
@@ -231,6 +250,18 @@ export class TaskFormCloudComponent implements OnInit, OnChanges, OnDestroy {
         this.executeOutcome.emit(outcome);
     }
 
+    switchToDisplayMode(newDisplayMode?: string) {
+        this.adfCloudForm.switchToDisplayMode(newDisplayMode);
```

Comment:
Maybe we could call the service directly here? e.g. instead of calling child component method?

```
this.cloudFormRenderingService(we can have separate service)
  .switchToDisplayMode(this.id, FormCloudDisplayMode[newDisplayMode], this.displayMode, this.displayModeConfigurations);
```

The `id` can be passed to `adf-cloud-form`, in that case we could manipulate the form from any place in our app, we would just need to have the service and `id` (other data should also be available 🤔), what do you think?